src/constants.js

```diff
@@ -0,0 +1,9 @@
+export const API_STATUS = {
+  IDLE: "idle",
+  LOADING: "loading",
+  SUCCESS: "success",
+  ERROR: "error",
+};
+export const API_URL = "https://dhrutham-connect-backend.herokuapp.com";
+// export const API_URL = "http://localhost:5000";
+export const CLOUDINARY_VIDEO_TRANSFORMATION_URL = `https://res.cloudinary.com/${process.env.CLOUD_NAME}/image/upload/w_100,h_100,c_thumb,r_max`;
```

Comment:
you can write in a different way so that the images will be converted into .webp format which is best for optimization. You can read about it on https://cloudinary.com/blog/transparent_webp_format_cdn_delivery_based_on_visitors_browsers